.travis.yml

```diff
@@ -1,12 +1,9 @@
 language: php
 
 php:
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
```

Comment:
It's questionable to me to remove 5.6 testing, when we still have 5.6 listed as a supported PHP version.